packages/contracts-bedrock/test/DeployOPChain.t.sol

```diff
@@ -336,9 +338,18 @@ contract DeployOPChain_TestBase is Test {
     uint32 basefeeScalar = 100;
     uint32 blobBaseFeeScalar = 200;
     uint256 l2ChainId = 300;
+    AnchorStateRegistry.StartingAnchorRoot[] startingAnchorRoots;
     OPStackManager opsm = OPStackManager(address(0));
 
     function setUp() public virtual {
+        // Set defaults for reference types
+        startingAnchorRoots.push(
+            AnchorStateRegistry.StartingAnchorRoot({
+                gameType: GameTypes.CANNON,
+                outputRoot: OutputRoot({ root: Hash.wrap(keccak256("defaultOutputRoot")), l2BlockNumber: 400 })
```

Comment:
This l2BlockNumber is arbitrary right?

Reply:
Yes, it's a continuation of the numbering above.